marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/core.sjs

```diff
@@ -43,7 +43,8 @@ function getTypesInfo() {
                 fileExtension: getArtifactFileExtension(artifactType),
                 directory: getArtifactDirectory(artifactType),
                 nameProperty: artifactLibrary.getNameProperty(),
-                versionProperty: artifactLibrary.getVersionProperty()
+                versionProperty: artifactLibrary.getVersionProperty(),
+                updateRole: artifactLibrary.getPermissions().filter((perm) => perm.capability === 'update').map((perm) => xdmp.roleName(perm.roleId))[0]
```

Comment:
Similar to https://github.com/marklogic/marklogic-data-hub/pull/3605, I'm wondering if it'd be better for the "roles" service to instead include true/false properties for e.g. "canLoadMappings", "canLoadFlows", etc. That prevents the Java tier from having knowledge of any roles. 

More importantly, ML knows the answer to the question of whether a user can load a mapping, so it should be the one to determine a true/false value for that, not the Java code.